projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1850,10 +1849,14 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
                 case GridColumnDataType.DateTime:
                     this.filters = IgxDateTimeFilteringOperand.instance();
                     break;
+                case GridColumnDataType.Image:
+                    this.filterable = false;
```

Comment:
As a user, I am able to make the image column sortable, but this line prevents me from setting it as filterable, not sure if expected.

Reply:
Expand the lines above and you'll see this is executed only if there's no filters provided.